locale/en-US/zotero-better-bibtex.dtd

```diff
@@ -106,15 +106,16 @@ https://github.com/retorquere/zotero-better-bibtex/issue">
 <!ENTITY better-bibtex.Preferences.prefpane.better-bibtex "Better BibTeX">
 <!ENTITY better-bibtex.Preferences.rescan-citekeys "Re-scan citekeys">
 <!ENTITY better-bibtex.Preferences.resetCache "Reset cache">
-<!ENTITY better-bibtex.BetterBibTeX.reportErrors "Report Better BibTeX Errors...">
-<!ENTITY better-bibtex.BetterBibTeX.auxScanner "Scan BibTeX AUX file for references...">
+<!ENTITY better-bibtex.BetterBibTeX.reportErrors "Report Better BibTeX Errors…">
+<!ENTITY better-bibtex.BetterBibTeX.auxScanner "Scan BibTeX AUX file for references…">
 <!ENTITY better-bibtex.bulk-keys-confirm.message "You are changing more than {treshold} items in one go. Are you sure?">
 <!ENTITY better-bibtex.bulk-keys-confirm.whatever "OK, and don't ask me again">
+<!ENTITY better-bibtex.BetterBibTeX.menu "BibTeX options">
```

Comment:
I'd prefer this to be named "Better BibTeX" to avoid confusion with regular Zotero BibTeX handling